kong.conf.default

```diff
@@ -1213,12 +1213,34 @@
 # See the lua-nginx-module documentation for more information:
 # https://github.com/openresty/lua-nginx-module
 
-#lua_ssl_trusted_certificate =   # Absolute path to the certificate
-                                 # authority file for Lua cosockets in PEM
-                                 # format. This certificate will be the one
-                                 # used for verifying Kong's database
-                                 # connections, when `pg_ssl_verify` or
-                                 # `cassandra_ssl_verify` are enabled.
+
+#lua_ssl_trusted_certificate =   # Comma-separated list of paths to certificate
+                                 # authority files for Lua cosockets in PEM format.
+                                 #
+                                 # These certificates will be the combined
+                                 # into a single file during the startup phase.
+                                 #
+                                 # The special value `system` expands to the
+                                 # "usual default" provided by each distro. The following
+                                 # paths will be tested in order, and the first one found
+                                 # will be used:
```

Comment:
```suggestion
                                 # The special value `system` expands to the
                                 # "usual default" provided by each distro, according
                                 # to an arbitrary heuristic. In the current implementation,
                                 # The following pathnames will be tested in order,
                                 # and the first one found will be used:
```

Sorry about the bit of "language lawyering", just trying to keep the door open so we can adjust the automagic behavior if we deem necessary in the future without incurring in a breaking change. :)